kolibri/plugins/learn/assets/src/views/content-card/index.vue

```diff
@@ -1,18 +1,23 @@
 <template>
 
-  <router-link :to="link" class="card" :class="{ 'mobile-card': isMobile }">
-
-    <card-thumbnail
-      class="thumbnail"
-      :thumbnail="thumbnail"
-      :kind="kind"
-      :progress="progress"
-      :isMobile="isMobile"
-    />
-
-    <h3 class="text" dir="auto">{{ title }}</h3>
-
-  </router-link>
+  <span ref="complete-title">
```

Comment:
> There's a small problem after wrapping the `<router-link>` with a `<span>` (though I'm not quite sure if this is the best practice for adding tooltips) since it's causing a problem with the bottom margin for the content cards.

Why was this necessary? Why not add the `ref` directly to the `router-link` component?

Reply:
I did something like this for my first attempt

```
  <router-link :to="link" class="card" :class="{ 'mobile-card': isMobile }" ref="complete-title">
    <card-thumbnail
      class="thumbnail"
      :thumbnail="thumbnail"
      :kind="kind"
      :progress="progress"
      :isMobile="isMobile"
    />
    <h3 class="text" dir="auto" v-shave>{{ title }}</h3>
    <ui-tooltip v-if="hasTooltip" trigger="complete-title">
      {{ title }}
    </ui-tooltip>
  </router-link>
```

But it was having a weird output so I thought I needed to wrap it inside an element
![screen shot 2018-03-05 at 12 27 55 pm](https://user-images.githubusercontent.com/27984604/36957640-b0ef8820-2070-11e8-9bd4-baa3c0bd7541.png)